tigris/collection.go

```diff
@@ -193,6 +193,22 @@ func (c *Collection[T]) Read(ctx context.Context, filter filter.Filter, fields .
 	return &Iterator[T]{Iterator: it}, nil
 }
 
+// Read returns documents which satisfies the filter.
+// Only field from the give fields are populated in the documents. By default, all fields are populated.
+func (c *Collection[T]) Explain(ctx context.Context, filter filter.Filter, fields ...*fields.Read) (*driver.ExplainResponse, error) {
```

Comment:
We may want to not expose driver.ExplainResponse, just define a type alias as others endpoint do.